app.js

```diff
@@ -94,6 +96,13 @@ applyProxy(app, {
   rewrite: false
 });
 
+app.use('/em-anno', proxy({
```

Comment:
Can we use applyProxy here to be consistent with other usages?

Reply:
Andrew - As discussed, due to the pathRewrite value, I am using app.use function as is to avoid any further complexities.

Reply:
As discussed it is up to you but I see it as a quick win. Here are some details which I already tested locally and worked fine:
```
  if (false !== config.rewrite) {
    options.pathRewrite = {
        [`^${config.source}`]: config.rewriteUrl || ''
    };
  }
```
and then:
```
applyProxy(app, {
  source: '/em-anno',
  target: config.get('proxy.mv_annotations'),
  rewrite: false,
  rewriteUrl: '/api'
});
```

Reply:
done